core/tests/puppeteer-acceptance-tests/blog-post-draft-blogs.js

```diff
@@ -0,0 +1,16 @@
+const e2eBlogPostAdmin = require("./utility-functions/blogAdminUtils.js");
+const testConstants = require("./utility-functions/testConstants.js");
+
+const blogDashboard = testConstants.URLs.BlogDashboard;
+
+async function deleteDraftAsBlogAdmin() {
+  const blogPostAdmin = await new e2eBlogPostAdmin();
+  await blogPostAdmin.getInitialized();
+
+  await blogPostAdmin.signInWithEmail("testadmin@example.com");
+  await blogPostAdmin.goto(blogDashboard);
+  await blogPostAdmin.deleteDraftBlog();
```

Comment:
How do we know there's a draft blog to delete? Wouldn't that need to be created first?

Also note that "blog" refers to the whole blog; you probably want "blog post" here. Ditto elsewhere.

Reply:
done